LondonBritishLibrary/orient/BLorient501.xml

```diff
@@ -294,39 +295,83 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
                                     <ab type="ruling"> The upper line is written above the ruling,
                                         the lower line is written above the ruling.</ab>
-                                    <ab type="CruxAnsata">
-                                        Most pages are decorated with several crux ansata with coronis.
-                                    </ab>
-                                    <ab type="coronis">
-                                        Most pages are decorated with several crux ansata with coronis.
-                                    </ab>
-                                </layout>
-                                
+                                    <ab type="CruxAnsata">Most pages are decorated with several crux ansata with coronis.</ab>
+                                    <ab type="coronis">Most pages are decorated with several crux ansata with coronis.</ab>
+                                </layout>     
                             </layoutDesc>
                         </objectDesc>
                         <handDesc>
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>Fine hand of the <date>15th century</date>.
                                 </desc>
                                 <seg type="ink">Black, red</seg>
+                                <seg type="rubrication">Crux ansata, headlines, numbers and punctuation marks</seg>
                                 <date notBefore="1400" notAfter="1500"/>
-                            </handNote>
-                            
-                                                        
+                            </handNote>                          
                         </handDesc>
-
                   <decoDesc>
                      <decoNote xml:id="d1" type="frame">
-                        <locus target="#1r #62r #70r #123r #177r"/>
-        The commencement of each book is ornamented in the usual style of the 15th century
-        
-    </decoNote>
+                         <locus target="#1r #62r #70r #79v #97r #123r #177r"/>
+                         <desc>The commencement of each book is decorated with a frame in the usual 15th century style.</desc>
```

Comment:
"each book" would mean every book in the manuscript, it seems to me that there are some books that have an incipit page with a frame and some books that have just a band. so either the decoNotes should be split one by one 
(i mean 
```
<decoNote xml:id="d1" type="frame">
<locus target="#1r"/>
                         <desc>The incipit of <ref type="item" target="#ms_i1"/> is decorated with a frame in the usual 15th century style.</desc>
</decoNote>
<decoNote xml:id="d2" type="frame">
<locus target="#62r"/>
                         <desc>The incipit of <ref type="item" target="#ms_i2"/> is decorated with a frame....</desc>
</decoNote>
```
... etc etc)
or "each book" should be replaced in both cases by "some of the books"

Reply:
Yes, the commencement of each Biblical book has either a frame or a band. I changed to:
"The commencements of Biblical books are decorated with a frame in the usual 15th century style."
"The commencements of Biblical books are decorated with a band in the usual 15th century style."